src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -65,6 +69,8 @@ public CommentAttributes(Comment comment) {
         this.recipients = comment.getRecipients();
         this.createdAt = comment.getCreatedAt();
         this.commentText = comment.getCommentText();
+        this.lastEditorEmail = comment.getLastEditorEmail();
+        this.lastEditedAt = comment.getLastEditedAt();
```

Comment:
this 2 attributes will be null for legacy comments and 
will be dangerous for other developers. it needs a default value instead